packages/js/product-editor/src/components/iframe-editor/header-toolbar/header-toolbar.scss

```diff
@@ -1,39 +1,40 @@
 .woocommerce-iframe-editor__header-toolbar {
-    height: 60px;
-    border: 0;
-    border-bottom: 1px solid $gray-400;
-    display: flex;
-    align-items: center;
+	height: 60px;
+	border: 0;
+	border-bottom: 1px solid $gray-400;
+	display: flex;
+	align-items: center;
 	justify-content: space-between;
 
-    .woocommerce-iframe-editor__header-toolbar-inserter-toggle.components-button.has-icon {
-        height: 32px;
-        margin-right: 8px;
-        min-width: 32px;
-        padding: 0;
-        width: 32px;
+	.woocommerce-iframe-editor__header-toolbar-inserter-toggle.components-button.has-icon {
+		height: 32px;
+		margin-right: 8px;
+		min-width: 32px;
+		padding: 0;
+		width: 32px;
 
-        svg {
-            transition: transform .2s cubic-bezier(.165,.84,.44,1);
-        }
+		svg {
+			transition: transform 0.2s cubic-bezier(0.165, 0.84, 0.44, 1);
+		}
 
-        &.is-pressed:before {
-            width: 100%;
-            left: 0;
-        }
+		&.is-pressed:before {
+			width: 100%;
+			left: 0;
+		}
 
-        &.is-pressed svg {
-            transform: rotate(45deg);
-        }
-    }
+		&.is-pressed svg {
+			transform: rotate(45deg);
+		}
+	}
 
-    &-left {
-        padding-left: $gap-small;
-    }
+	&-left {
+		padding-left: $gap-small;
+	}
 	&-right {
 		display: flex;
 		justify-content: center;
 		align-items: center;
+		gap: $gap-smaller;
```

Comment:
This fixed a small issue with the `cancel`/`Done` fields, which got smashed together if you shrunk the page to a small view size and increased it again.